platforms/springboot/src/main/java/io/hawt/springboot/HawtioManagementConfiguration.java

```diff
@@ -34,23 +35,21 @@
 import io.hawt.web.filters.XXSSProtectionFilter;
 import io.hawt.web.proxy.ProxyServlet;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.actuate.autoconfigure.jolokia.JolokiaEndpoint;
-import org.springframework.boot.actuate.autoconfigure.jolokia.JolokiaEndpointAutoConfiguration;
-import org.springframework.boot.actuate.autoconfigure.web.ManagementContextConfiguration;
 import org.springframework.boot.autoconfigure.AutoConfigureAfter;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
 import org.springframework.boot.web.servlet.FilterRegistrationBean;
 import org.springframework.boot.web.servlet.ServletContextInitializer;
 import org.springframework.boot.web.servlet.ServletListenerRegistrationBean;
 import org.springframework.boot.web.servlet.ServletRegistrationBean;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.core.Ordered;
 import org.springframework.web.servlet.handler.SimpleUrlHandlerMapping;
 import org.springframework.web.servlet.mvc.AbstractUrlViewController;
 
 import static io.hawt.web.filters.BaseTagHrefFilter.PARAM_APPLICATION_CONTEXT_PATH;
 
-@ManagementContextConfiguration
+@Configuration
```

Comment:
Isn't this the reason why we have the issue with the management feature?  If `@ManagementContextConfiguration` isn't simply available at SB 3, maybe we can track the changes of the deletion of the annotation to see why it's removed and how we should rewrite it for our app.

Reply:
Looks like we had a problem because I was missing imports for management as you mentioned. https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/actuate/autoconfigure/web/ManagementContextConfiguration.html I've tried with also with `@Configuration` only and suddenly it works